profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -225,6 +226,20 @@ function tmgmt_poetry_form_tmgmt_job_form_alter(&$form, &$form_state, $form_id)
 }
 
 /**
+ * Custom validator to add a server validation step for dates in the past.
+ */
+function _tmgmt_poetry_validate($form, &$form_state) {
+  $delai = $form_state['values']['settings']['delai'];
+  if (is_string($delai)) {
+    $date = strtotime($delai);
+    $today = time();
+    if ($date < $today) {
+      form_set_error('delai', t('The date you selected is incorrect!'));
```

Comment:
Something similar here, I think it would be better to add a more explicit message: "The expected requested delivery date can't be in the past."